CMakeLists.txt

```diff
@@ -278,7 +278,7 @@ endif()
 if(UNIX AND NOT APPLE)
     execute_process(COMMAND grep -w "NAME" /etc/os-release OUTPUT_VARIABLE LINUX_DISTRO)
 
-    if(${LINUX_DISTRO} MATCHES "NAME=\"(Arch|Manjaro|Artix) Linux\"\n")
+    if(${LINUX_DISTRO} MATCHES "NAME=\"(Arch|Manjaro|Artix) Linux\"|NAME=\"openSUSE Tumbleweed\"\n")
```

Comment:
can you change this so that it is just NAME="(all the distros)" instead of an | on the outside, will make this simpler

Reply:
Unlike other distros, openSUSE does not get the "Linux" name after the distro name

Is that what you're talking about?